src/Apps/Conversation/Components/DetailsHeader.tsx

```diff
@@ -54,8 +63,16 @@ export const DetailsHeader: FC<DetailsHeaderProps> = props => {
 
 export const DetailIcon: React.FC<DetailsProps> = props => {
   const { showDetails, setShowDetails } = props
+  const isXL = __internal__useMatchMedia(themeProps.mediaQueries.xl)
```

Comment:
Dito here - does this break SSR rendering?

Reply:
I compared SSR on both (my changes locally and the current version on staging) and the behavior is the same. Though, I'm not sure if that's an accurate test because the sidebar only opens client-side 🤔

Reply:
If the sidebar is only open on the client, then no prob! Just had to call it out 🙂  

In any case, the updated implementation with Media makes sense 👍